mne/viz/tests/test_3d.py

```diff
@@ -102,3 +103,36 @@ def test_plot_trans():
     assert_raises(ValueError, plot_trans, evoked.info, trans_fname=trans_fname,
                   subject='sample', subjects_dir=subjects_dir,
                   ch_type='bad-chtype')
+
+
+def test_limits_to_control_points():
+    """Test helper function for determining control points from value or
```

Comment:
this should be one line, otherwise it won't be printed correctly when you run the tests with the `verbose` option